src/events/index.spec.js

```diff
@@ -614,4 +614,141 @@ describe('Events', function() {
       });
     });
   });
+
+  describe('subscribeEvent', function() {
+    context('when all the required parameters are provided', function() {
+      let expectedSubscription;
+      let promise;
+      let request;
+      let event;
+      let user;
+      let subscriptionFromServer;
+      let toCamelCase;
+
+      beforeEach(function() {
+        event = fixture.build('event');
+        user = fixture.build('contxtUser');
+
+        expectedSubscription = fixture.build('userEventSubscription');
+        subscriptionFromServer = fixture.build(
+          'userEventSubscription',
+          expectedSubscription,
+          {
+            fromServer: true
+          }
+        );
+
+        request = {
+          ...baseRequest,
+          post: sinon.stub().resolves(subscriptionFromServer)
+        };
+        toCamelCase = sinon
+          .stub(objectUtils, 'toCamelCase')
+          .callsFake((app) => expectedSubscription);
+
+        const events = new Events(baseSdk, request);
+        events._baseUrl = expectedHost;
+
+        promise = events.subscribeEvent(user.id, event.id);
+      });
+
+      it('posts the user event to the server', function() {
```

Comment:
I'd update this description to be something like `creates the user event subscription` because, as documentation, there is no user event being sent to the server

Reply:
👍